android/sdk/src/main/java/com/taobao/weex/ui/component/WXScroller.java

```diff
@@ -798,19 +802,24 @@ public void scrollTo(WXComponent component, Map<String, Object> options) {
     int viewXInScroller = 0;
     if (this.isLayoutRTL()) {
       // if layout direction is rtl, we need calculate rtl scroll x;
-      if (getInnerView().getChildCount() > 0) {
-        int totalWidth = getInnerView().getChildAt(0).getWidth();
-        int displayWidth = getInnerView().getMeasuredWidth();
-        viewXInScroller = totalWidth - (component.getAbsoluteX() - getAbsoluteX()) - displayWidth;
+      if (component.getParent() != null && component.getParent() == this) {
```

Comment:
Any reason to add this `if`?

Reply:
If hit this ```if```, it mean's scroll to indirect child element, then the offsetX calculate will different from direct child element